api/db-cdr/generate-cdr/libproject/genomic_manifests.rb

```diff
@@ -374,9 +382,55 @@ def _build_copy_manifest_row(
   }
 end
 
-def build_manifests_for_aw4_section(input_section, ingest_bucket, dest_bucket, display_version_id, aw4_rows, output_manifest_path)
-  # TODO(RW-8269): handle delta directories
-  path_prefix = "pooled/#{input_section["pooledDestPathInfix"]}/#{display_version_id}_base"
+# Logic to read the previous manifest for a delta release.
+# Returns a hash of person_id => CSV row found in previous manifest
+def _read_previous_manifest(project, dest_bucket, deltaReleaseManifestPath, deltaRelease, infix)
+  common = Common.new
+  prev_manifest = ""
+  manifest_path = deltaReleaseManifestPath
```

Comment:
should this be moved inside the `nil?` check?

Reply:
I wanted it here to be visible in line 414 
`raise ArgumentError.new("failed to read previous manifest from #{manifest_path},`

if deltaReleaseManifestPath is nil, then maifest_path will be nil, but it should be filled later in line 405. Overall I don't think it needs to be moved inside the nil check.